HISTORY.md

```diff
@@ -13,6 +13,7 @@
 * Fixed a BackupEngine bug in which RestoreDBFromLatestBackup would fail if the latest backup was deleted and there is another valid backup available.
 * Fix L0 file misorder corruption caused by ingesting files of overlapping seqnos with memtable entries' through introducing `epoch_number`. Before the fix, `force_consistency_checks=true` may catch the corruption before it's exposed to readers, in which case writes returning `Status::Corruption` would be expected. Also replace the previous incomplete fix (#5958) to the same corruption with this new and more complete fix.
 * Fixed a bug in LockWAL() leading to re-locking mutex (#11020).
+* Fixed a bug that `RefitLevel()` can move files from wrong level when base level is different from the last level under universal compaction and `Options::level_compaction_dynamic_level_bytes > 0`.
```

Comment:
"universal compaction and `Options::level_compaction_dynamic_level_bytes > 0`" looks like a contradiction. I didn't see universal compaction mentioned in the description, title, or linked discussion.

Reply:
universal compaction was a typo from me, supposed to be level compaction